core/shared/src/main/scala/scalaz/zio/IO.scala

```diff
@@ -491,13 +491,14 @@ sealed abstract class IO[+E, +A] extends Serializable { self =>
    */
   final def retryOrElse0[E1 >: E, S, E2, B](
     policy: Schedule[E1, S],
-    orElse: (E1, S) => IO[E2, B]
+    orElse: (E1, S) => IO[E2, B],
```

Comment:
In addition to `retryOrElse0`, there are two other methods that delegate to this one: `retryOrElse`, and `retry`. Both of these methods should take a `Clock` which defaults to `Clock.Live`.

Reply:
These are now (also) solved by [this commit](https://github.com/scalaz/scalaz-zio/pull/332/commits/5fa247a94df8893f299bc2a415484e90bf9ce81a)